client/blocks/recurring-payments-stats-nudge/index.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import isRecurringPaymentsStatsNudgeDismissed from 'state/selectors/is-recurring-payments-stats-nudge-dismissed';
+import QueryPreferences from 'components/data/query-preferences';
+import SectionHeader from 'components/section-header';
+import { dismissNudge } from './actions';
+import { enhanceWithSiteType, recordTracksEvent } from 'state/analytics/actions';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { withEnhancers } from 'state/utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class RecurringPaymentsStatsNudge extends Component {
+	static propTypes = {
+		isDismissed: PropTypes.bool.isRequired,
+		plan: PropTypes.object,
+		recordTracksEvent: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		translate: PropTypes.func.isRequired,
+		primaryButton: PropTypes.bool.isRequired,
+	};
+
+	componentDidMount() {
+		this.recordView();
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.siteId && this.props.siteId && this.props.siteId !== prevProps.siteId ) {
+			this.recordView();
+		}
+	}
+
+	recordView() {
+		if ( this.isVisible() ) {
+			this.props.recordTracksEvent( 'calypso_recurring_payments_stats_nudge_view' );
+		}
+	}
+
+	recordClick = eventName => {
+		const plan = this.props.plan ? this.props.plan.productSlug : '';
+
+		this.props.recordTracksEvent( eventName, { plan } );
+	};
+
+	onDismissClick = () => {
+		this.recordClick( 'calypso_recurring_payments_stats_nudge_dismiss_icon_click' );
+		this.props.dismissNudge();
+	};
+
+	onStartNowClick = () => {
+		this.recordClick( 'calypso_recurring_payments_stats_nudge_start_now_button_click' );
+	};
+
+	isVisible() {
+		return ! this.props.isDismissed;
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		if ( ! this.isVisible() ) {
+			return null;
+		}
+
+		return (
+			<Card className="recurring-payments-stats-nudge">
+				<QueryPreferences />
+
+				<Gridicon
+					icon="cross"
+					className="recurring-payments-stats-nudge__close-icon"
+					onClick={ this.onDismissClick }
+				/>
+
+				<SectionHeader
+					className="recurring-payments-stats-nudge__header"
+					label={ translate( 'Recommendations from WordPress.com' ) }
+				/>
+
+				<div className="recurring-payments-stats-nudge__body">
+					<div className="recurring-payments-stats-nudge__image-wrapper">
+						<img
+							className="recurring-payments-stats-nudge__image"
+							src="/calypso/images/earn/earn-section.svg"
```

Comment:
After #36863 was merged, I'm not sure that this will work after merged as the image is now here: https://github.com/Automattic/wp-calypso/blob/master/client/assets/images/earn/earn-section.svg

It's probably necessary to import the image instead (if it's going to be used).